FloripaBus/FloripaBus.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+
+using Xamarin.Forms;
+using Ninject;
+using Ninject.Modules;
+
+namespace FloripaBus
+{
+	public class App : Application
+	{
+		public static StandardKernel Container { get; set; }
+
+		public App ()
+		{
+			Ninja.LoadModule (new SharedModule ());
```

Comment:
Ninject handles the DI and IoC easily. There is no much setup to do.
It uses fluent interface for set up which I really like.
Using the Ninject modules we can create different injections for Android and iOS (for example) easily and readily.